src/interfaces/root-state-interface.ts

```diff
@@ -0,0 +1,14 @@
+import { CurrentUserIF, CurrentAccountIF } from '@/interfaces'
+
+/** The ROOT state model interface. */
+export interface RootStateIF {
+  account: {
+    currentUser: CurrentUserIF,
+    currentAccount: CurrentAccountIF
+  },
```

Comment:
You probably don't need this object, since you don't use it in the getters. Do you think it will be used in the future?

Reply:
Let's keep it simple. Thanks!